token/cli/src/main.rs

```diff
@@ -7425,4 +7453,43 @@ mod tests {
                 .unwrap()
         );
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn metadata_pointer() {
+        let (test_validator, payer) = new_validator_for_test().await;
+        let config =
+            test_config_with_default_signer(&test_validator, &payer, &spl_token_2022::id());
+        let program_id = &spl_token_2022::id();
```

Comment:
nit: you can reuse the same variable, no need to redeclare it elsewhere!
```suggestion
        let program_id = &spl_token_2022::id();
        let config =
            test_config_with_default_signer(&test_validator, &payer, &program_id);
```